src/OpenDebugAD7/MILaunchOptions.cs

```diff
@@ -581,6 +604,22 @@ private static string GetLLDBMIPath()
                     xmlLaunchOptions.Append("    </Environment>\n");
                 }
 
+                if (jsonLaunchOptions.SymbolLoadInfo != null)
+                {
+                    xmlLaunchOptions.Append("    <SymbolLoadInfo ");
+                    if (jsonLaunchOptions.SymbolLoadInfo.LoadAll.HasValue)
+                    {
+                        xmlLaunchOptions.Append("LoadAll = '" + jsonLaunchOptions.SymbolLoadInfo.LoadAll.Value.ToString().ToLowerInvariant() + "' ");
+                    }
+
+                    if (!String.IsNullOrWhiteSpace(jsonLaunchOptions.SymbolLoadInfo.ExceptionList))
+                    {
+                        xmlLaunchOptions.Append("ExceptionList='" + XmlSingleQuotedAttributeEncode(jsonLaunchOptions.SymbolLoadInfo.ExceptionList) + "' ");
```

Comment:
can you validate that if individual strings are quoted in there that it works properly?

say my exclusionList is `"/foo bar/fooobar.so";"/baz baz/barfoo.so"` or some combination that we pass the right things to `gdb` ?

Reply:
If you add quotes, you are looking for the library name foobar.so", not foobar.so. I think it is filenames, and not paths so users should not be quoting between semicolons.